event.js

```diff
@@ -0,0 +1,45 @@
+var button = document.querySelector('#newsButton');
+button.addEventListener('click', doSomething);
+
+function doSomething() {
+  //if my <p>is showing hide it,else show it
+  var news = document.querySelector('#newsInfo');
+  if (news.style.display === 'none') {
+    news.style.display = 'block';
+  } else {
+    news.style.display = 'none';
+  }
+
+}
+
+
+var xhr = new XMLHttpRequest();
+xhr.onreadystatechange = function () {
+  if (xhr.readyState === 4) {
+    if (xhr.status === 200) {
```

Comment:
You can combine these two if statements into one by using ' && ' 

For example:

https://www.w3schools.com/js/tryit.asp?filename=tryjs_comparison_and